docs/cluster_up_down.md

```diff
@@ -0,0 +1,204 @@
+  ```
+
+### Linux
+
+1. Install Docker with your platform's package manager. 
+2. Configure the Docker daemon with an insecure registry parameter of `172.30.0.0/16`
+   In RHEL and Fedora, edit the `/etc/sysconfig/docker` and add or uncomment the following line:
+
+   ```
+   INSECURE_REGISTRY='--insecure-registry 172.30.0.0/16'
+   ```
+   After editing the config, restart the Docker daemon.
+
+3. Download the Linux `oc` binary from 
+   [openshift-origin-client-tools-VERSION-linux-64bit.tar.gz](https://github.com/openshift/origin/releases/latest) 
+   and place it in your path.
+4. Open a terminal with a user that has permission to run Docker commands and run:
+   ```
+   $ oc cluster up
+   ```
+
+To stop your cluster, run:
+```
+$ oc cluster down
+```
+
+## Administrator Access
+
+To execute administrator commands on your cluster, `docker exec` into the `origin` container:
+
+```
+docker exec -ti origin bash
+```
+
+## Docker Machine
+
+By default, when `--create-machine` is used to create a new Docker machine, the `oc cluster up` command will use the  
+VirtualBox driver. In order to use a different driver, you must create the Docker machine beforehand
+and either specify its name with `--docker-machine` argument, or set its environment using the `docker-machine env`
+command. When creating a Docker machine manually, you must specify the `--engine-insecure-registry` secure registry 
+parameter expected by OpenShift.
+
+Following are examples of creating a new Docker machine in OS X using the [xhyve](https://github.com/zchee/docker-machine-driver-xhyve) driver, 
+and in Windows, using the `hyperv` driver.
+
+OS X:
+```
+$ docker-machine create --driver xhyve --engine-insecure-registry 172.30.0.0/16 mymachine
+```
+
+Windows (running a command window as Administrator):
+```
+c:\> docker-machine create --driver hyperv --engine-insecure-registry 172.30.0.0/16 mymachine
+```
+
+When the `--docker-machine` argument is specified on `oc cluster up`, the machine's environment does not need to be configured
+on the current shell. Also if the machine exists but is not started, `oc cluster up` will attempt to start it.
+
+## Configuration
+
+`oc cluster up` creates its configuration in `/var/lib/origin/openshift.local.config` on the Docker host.
+To specify a different location for it, use the `--host-config-dir` argument. 
+
+A new configuration will be generated by default each time the cluster is started. To make changes to the configuration and 
+preserve those changes, use the `--use-existing-config` argument when starting your cluster.
+
+To make a local copy of the configuration you can use Docker cp:
+
+```
+docker cp origin:/var/lib/origin/openshift.local.config .
+```
+
+## Etcd Data
+
+If a host data directory is not specified, the data directory used by OpenShift is discarded when the container is destroyed.
+
+To persist data across restarts, specify a valid host directory in the `--host-data-dir` when starting your cluster. As long
```

Comment:
"specify a valid host directory using the `--host-data-dir` argument when starting your cluster (with cluster up?)"